Utilities/StrFmt.cpp

```diff
@@ -612,14 +612,24 @@ std::vector<std::string> fmt::split(std::string_view source, std::initializer_li
 
 std::string fmt::trim(const std::string& source, std::string_view values)
 {
-	usz begin = source.find_first_not_of(values);
+	const usz begin = source.find_first_not_of(values);
 
 	if (begin == source.npos)
 		return {};
 
 	return source.substr(begin, source.find_last_not_of(values) + 1);
 }
 
+std::string fmt::trim_front(const std::string& source, std::string_view values)
```

Comment:
I think it's better to return a string view, or at least have two functions like fs::get_parent_dir_view

Reply:
Well, if it ever is necessary we can add it. But for this PR it hardly matters at all